.travis.yml

```diff
@@ -0,0 +1,22 @@
+language: go
+
+services:
+  - docker
+
+go:
+  - 1.7
+
+env:
+  - CGO_ENABLED=0
+
+script:
+  - go vet
+  - go test
+  - go build -ldflags "-s -w -X main.build=$TRAVIS_BUILD_NUMBER -X main.rev=$TRAVIS_COMMIT" -a -tags netgo
```

Comment:
I think `-X main.build=$TRAVIS_BUILD_NUMBER` is unnecessary, no `build` variable in main package.

That command is for building static binaries in earlier versions of go.

Try: `go build -a -ldflags "-X main.rev=$REV"`